exercises/sum-of-multiples/example.py

```diff
@@ -1,4 +1,7 @@
 def sum_of_multiples(limit, multiples):
-    return sum(value for value in range(limit)
+    if 0 in multiples:
+        return 0
+    else:
+        return sum(value for value in range(limit)
                if any(value % multiple == 0
```

Comment:
Needs more indent on line 6.

```suggestion
                   if any(value % multiple == 0
```